src/server/scripts/EasternKingdoms/SunwellPlateau/boss_kiljaeden.cpp

```diff
@@ -482,6 +482,20 @@ struct boss_kiljaeden : public BossAI
         }
     }
 
+    void JustSummoned(Creature* summon) override
+    {
+        summons.Summon(summon);
```

Comment:
You can call the base class method `BossAI::JustSummoned(summon);` which calls `summons.Summon` 

I'd also refactor only to add this 1 line to make the diff smaller